CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
   * 3.2.0-rc1 [\#5280](https://github.com/rvm/rvm/pull/5280)
   * 3.2.0 [\#5283](https://github.com/rvm/rvm/pull/5283)
   * 3.2.1 [\#5306](https://github.com/rvm/rvm/pull/5306)
+  * 3.0.5, 3.1.3 and 3.2.1 For Ubuntu 22.10 / x86_64 [\#5308](https://github.com/rvm/rvm/pull/5308)
```

Comment:
The binary section is a bit below, called "Binaries". Please move to that section.

Reply:
done